### PR TITLE
Announcing Debezium UI PoC

### PR DESCRIPTION
**Pull request by gunnarmorling**
@rk3rn3r, @bibryam, feedback welcome!

### CODE REVIEW

File: blog/2020-10-22-towards-debzium-ui.adoc [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+There could be views for monitoring and trouble-shooting connectors, and much more.
+
+But before further progressing with this, we'd like to gather your feedback and opinions:
+Do you consider a graphical UI for Debezium useful in general, and is it something you would use in your projects?
+What is your feedback on the functionality currently implemented in the PoC?
+Which other functionality besides connector configuration would you like to see in a Debezium UI?
+We've provided a short survey with these and a few other questions:
+
+[.text-center]
+https://docs.google.com/forms/d/e/1FAIpQLSfEEqslTWSLX89gzIDmSE_4v8hH0mYg0YBRaXhfDrrBbCUJgQ/viewform?usp=sf_link[Go to survey]
```
Comment (gunnarmorling) [inline blog/2020-10-22-towards-debzium-ui.adoc:46]: @rk3rn3r, please see the live questionnaire for how I ended up phrasing your option around guided configuration.

File: blog/2020-10-22-towards-debzium-ui.adoc [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+The scope of the PoC is the set-up flow for configuring and instantiating a Debezium Postgres connector.
+The user is guided through the required configuration steps in a wizard interface,
+starting from mandatory information (e.g. database credentials), over selecting the tables to be captured, up to optional settings like different data mapping options.
+After reviewing the final configuration, the UI will instantiate the connector in Kafka Connect.
+
+You can see a short demo of how this looks like in this video:
+
+++++
+<div class="responsive-video">
+<iframe width="1600" height="900" src="https://www.youtube.com/embed/RZ_3DF7Ndnk" frameborder="0" allowfullscreen></iframe>
```
Comment (rk3rn3r) [inline blog/2020-10-22-towards-debzium-ui.adoc:27]: 1600 is a very big screen

**Comment by rk3rn3r** [comment]
I like :+1:

File: blog/2020-10-22-towards-debzium-ui.adoc (hunk 2 above)
Comment (rk3rn3r) [reply blog/2020-10-22-towards-debzium-ui.adoc:27]: Is this the correct version of the video? There's a small "slip of the tongue" in the first sentences where you are talking about the Debezium team that were building it.

Comment (gunnarmorling) [reply blog/2020-10-22-towards-debzium-ui.adoc:27]: > 1600 is a very big screen

This the _maximum_, it's embedded into a responsive div.

> Is this the correct version of the video?

It's the only one I have. It's not perfect, but I thought it's good enough. At which timestamp is this glitch you mean?

Comment (rk3rn3r) [reply blog/2020-10-22-towards-debzium-ui.adoc:27]: 0:07

**Pull request merged**